liberty-maven-webapp-parent/pom.xml

```diff
@@ -17,32 +17,44 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
     
-    <parent>
-        <groupId>net.wasdev.wlp.maven</groupId>
-        <artifactId>liberty-maven</artifactId>
-        <version>2.0-SNAPSHOT</version>
-    </parent>
-    
     <groupId>net.wasdev.wlp.maven.parent</groupId>
     <artifactId>liberty-maven-webapp-parent</artifactId>
     <version>2.0-SNAPSHOT</version>
     <packaging>pom</packaging>
+	
+	<name>Liberty Maven Plugin Parent</name>
+	<description>Liberty Maven Plugin Parent</description>
+	<url>https://github.com/WASdev/ci.maven</url>
+	
+	<licenses>
+		<license>
+			<name>The Apache Software License, Version 2.0</name>
+			<url>https://raw.github.com/WASdev/ci.maven/master/LICENSE</url>
+			<distribution>repo</distribution>
+		</license>
+	</licenses>
     
-    <name>Liberty Maven Plugin Parent</name>
-    <description>Liberty Maven Plugin Parent</description>
-    <url>https://github.com/WASdev/ci.maven</url>
+    <scm>
```

Comment:
minor indention problem with <scm> tag and there is a mixture of 4 space and tab indention.  The <licenses> and <scm> elements are using tabs. It looks like 8 spaces instead of 4.